inspire/base/templates/search/search.html

```diff
@@ -19,23 +19,9 @@
 
 {% extends "search/search_base.html" %}
 
+{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import mathjax %}
+
 {%- block javascript %}
 {{ super() }}
-<script src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.5.3/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
-<script type="text/javascript">
-    require([
-      "jquery",
-      ], function ($) {
-          $(document).ready(function () {
-            MathJax.Hub.Config({
-                tex2jax: {inlineMath: [['$', '$'], ['\\(', '\\)']],
-                processEscapes: true},
-                showProcessingMessages: false,
-                messageStyle: "none"
-            });
-            MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
-          })
-      }
-    );
-</script>
+{{ mathjax() | safe }}
```

Comment:
Yes, otherwise the output of the macro was not rendering correctly. I've updated the macro call on the detailed template to mark it safe as well.

Am I right to assume that we can automatic escaping  enabled by default in Jinja2? http://jinja.pocoo.org/docs/dev/templates/#working-with-automatic-escaping